EIPS/eip-4337.md

```diff
@@ -227,51 +227,61 @@ To help make sense of these params, note that a malicious paymaster can at most
 
 `eth_sendUserOperation`
 
+eth_sendUserOperation submits a User Operation object to the User Operation pool of the client. An entryPoint address `MUST` be specified, and the client `MUST` only simulate and submit the User Operation through the specified entryPoint.
+
```

Comment:
maybe mention:
result SHOULD be set to true only if the request was accepted into the mempool (i.e. passed simulation)

Reply:
might also add a SHOULD specify the failure reason (invalid entrypoint, wallet opcode failed, paymaster opcode failed, simulation reverted with "xxx")
this is just a SHOULD, since it is a "mere" helper for developers/integrators.
the user can perform simulation locally, but opcode validations is hard to verify locally

Reply:
good point, pushed another commit to fix this